react-app/src/components/UploadManager.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+
+import { 
+  TextField,
+  IconButton,
+  Typography
+} from '@material-ui/core'
+import ArrowUpwardIcon from '@material-ui/icons/ArrowUpward'
+
+class UploadManager extends React.Component {
+  constructor() {
+    super()
+    this.state = {
+      name: ''
+    }
+
+    this.onInputChange = this.onInputChange.bind(this)
+  }
+
+  onInputChange(event) {
+    this.setState({
+      [event.target.name]: event.target.value
+    })
+  }
+
+  onUploadClick = () => {
+    localStorage[this.state.name] = this.props.photo.dataUri
+    this.props.setPhotoTaken(false)
+  }
+
+  render() {
+    return (
+      <div style={{display: 'flex', padding: '12px 12px', alignItems: 'center'}}>
+        <span>
+          <label htmlFor='name'>
+            <Typography variant="h6" style={{padding: '0px 12px'}}>
+              Name:
+            </Typography>
+          </label>
+        </span>
+        <TextField
+          id='name'
+          name='name' 
+          type='text'
```

Comment:
This text field gives suggestions based on previous input, which may not be a feature we want. I think it just clutters the UI a bit.

I've never seen this before, so I think its from this `type='text'` field, but I'm not totally sure.

Reply:
For some reason I couldn't reproduce this issue but I went ahead and removed the line as you suggested so let me know if it's still having problems.